pegasus/helpers/section_api_helpers.rb

```diff
@@ -372,6 +381,17 @@ def to_hash()
     }
   end
 
+  def to_shallow_hash()
+    {
+        id:@row[:id],
```

Comment:
Okay.  I like those names better.  Are we okay with just the rename as a incremental step toward a better handling of access control on section records?  See the **TODO** in `DashboardSection.fetch_if_allowed`.